contracts/scripts/standard-bridge/DeployStandardBridge.s.sol

```diff
@@ -16,20 +16,23 @@ contract BridgeBase is Script {
     // Amount of ETH which must be allocated only to the contract deployer on mev-commit chain genesis.
     uint256 public constant DEPLOYER_GENESIS_ALLOCATION = type(uint256).max - 500 ether;
 
-    // Amount of ETH to initially fund the relayer account on both chains.
-    uint256 public constant RELAYER_INITIAL_FUNDING = 1 ether;
+    // Amount of ETH to initially fund the relayer account on the mev-commit chain.
+    uint256 public constant RELAYER_INITIAL_FUNDING_MEV_COMMIT_CHAIN = 300 gwei; // 256 gwei + buffer
 
-    // Amount of ETH to initially fund the oracle account on L1 chain.
-    uint256 public constant ORACLE_INITIAL_FUNDING = 1 ether;
+    // Amount of ETH to initially fund the oracle account on the mev-commit chain.
+    uint256 public constant ORACLE_INITIAL_FUNDING_MEV_COMMIT_CHAIN = 300 gwei; // 256 gwei + buffer
+
+    // Amount of ETH to initially fund the relayer account on L1.
+    uint256 public constant RELAYER_INITIAL_FUNDING_L1 = 1 ether;
```

Comment:
@shaspitz Why do we need this? L1 transactions are being done by the users directly right?

Reply:
To finalize a transfer from mev-commit chain back to L1, the relayer needs funds on L1 to call the finalize tx